docs/sources/tempo/operations/troubleshooting/metrics-generator.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Troubleshoot Metrics Generator
+menuTitle: Metrics Generator
```

Comment:
Sorry, I missed this before. We're using sentence case. Using what we have in the other metrics-generator doc, Ii'm adding a suggestion to make this lower case and hyphenated for the title and sentence case for the menuTitle. 

```suggestion
title: Troubleshoot metrics-generator
menuTitle: Metrics-generator
```